improver/cli/__init__.py

```diff
@@ -274,15 +283,25 @@ def with_output(wrapped, *args, output=None, compression_level=1, **kwargs):
             printed instead.
         compression_level (int):
             Will set the compression level (1 to 9), or disable compression (0).
-
+        least_significant_digit (int):
+            If specified, the data will be quantized to a number of significant digits, see
+            http://www.esrl.noaa.gov/psd/data/gridded/conventions/cdc_netcdf_standard.shtml
+            for details. When used with `compression level`, this will result in lossy
+            compression.
```

Comment:
According to the link this is:

> power of ten of the smallest decimal place in unpacked data that is a reliable value

Sadly, this is not the same as the number of [significant digits](https://en.wikipedia.org/wiki/Significant_figures) (they've chosen the name of that attribute very confusingly).

Reply:
I've altered it, thanks